main-app/app/htmlTemplates/gameboard.html

```diff
@@ -0,0 +1,17 @@
+<table id="gameboard" >
+    <tr>
+        <td class="player{{gameboard[0]}}" ng-click="gameboardTapped(0)"><img src="images/blank.png"></td>
```

Comment:
food for thought - this is repetitive, and a construct we haven't covered yet will simplify this code down for you...